src/script/components/VerificationBadge/VerificationBadges.tsx

```diff
@@ -103,9 +109,18 @@ export const UserVerificationBadges = ({
 }) => {
   const {is_verified: isProteusVerified} = useKoSubscribableChildren(user, ['is_verified']);
   const {deviceIdentities} = useUserIdentity(user.qualifiedId, groupId, isSelfUser);
-  const {MLSStatus} = useMLSStatus({identity: deviceIdentities ? deviceIdentities[0] : undefined, user});
 
-  return <VerificationBadges context="user" isProteusVerified={isProteusVerified} MLSStatus={MLSStatus} />;
+  const mlsStatuses = getMLSStatuses({
+    identities: deviceIdentities,
+    user,
+  });
+  const status = !mlsStatuses
+    ? undefined
+    : mlsStatuses.length > 0 && mlsStatuses.every(status => status === MLSStatuses.VALID)
+      ? MLSStatuses.VALID
+      : MLSStatuses.NOT_ACTIVATED;
```

Comment:
wait it can also be `expired` or `revoked` 🤔

Reply:
ah maybe at the user level we don't have such distinction, but then `not activated` seems wrong 🤔

Reply:
Yeah, we don't narrow down on a user level that much. We could have multiple devices with different statuses. What should we display then? In a case where one device is revoked and another is expired.

This piece of code is how it worked before. I agree that NOT_ACTIVATED feels wrong as well.
I think it has been used as a culprit to show the expired icon anyways in that cases.
If something goes wrong with the name validation I dont show any icon